design/general-progress-monitoring.md

```diff
@@ -143,7 +151,14 @@ In the current implementation, Restic backups will move data during the "InProgr
 future, it may be possible to combine a snapshot with a Restic (or equivalent) backup which would
 allow for data movement to be handled in the "WaitingForPluginOperations" phase,
 
-The next phase is either "Completed", "WaitingForPluginOperations", "Failed" or "PartiallyFailed".
+The next phase would be "WaitingForPluginOperations" for backups or restores which have unfinished
```

Comment:
Hmm. That's a good question. We're not currently calling cancel. It might be good to add that as an additional cleanup step, although I don't know that it's essential.

Reply:
Yes, it is not urgent, let me take this into the data movement design. And we may address it when we implement Velero data movement if it is not done by then.